cvat-core/src/guide.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2023 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { SerializedGuide } from './server-response-types';
+import { ArgumentError, DataError } from './exceptions';
+import PluginRegistry from './plugins';
+import serverProxy from './server-proxy';
+
+class AnnotationGuide {
+    public readonly id?: number;
+    public readonly taskId: number;
+    public readonly projectId: number;
+    public readonly createdDate?: string;
```

Comment:
Shouln't we use new approach with `#prop`?

Reply:
Done